### DISCUSSION TITLE
API封装问题

### DISCUSSION THREAD

**Discussion by PeifengRen** [zai-org/ChatGLM3#688] [Q&A] [answered]
请问大佬一个困惑。如果我将prompt不卸载 Role为System的字段下 而是与用户问题一起拼接在User字段下 这会对模型的问答准确性等性能产生影响吗？一直有个困惑。希望大佬指点迷津

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
会的，System的指令跟随更强
